### PR TITLE
Combine some conda and pip install steps

### PR DESCRIPTION
**Pull request by tashrifbillah**
* combined conda and pip install steps
* basically preserved Ryan's original sequence
* I have double checked that `nvidia-pyindex` , `nvidia-tensorrt` do not have any effect on this program

### CODE REVIEW

**Comment by tashrifbillah** [comment]
Happy to talk on Monday before merging.

File: env_build_commands.md [hunk 1]
```diff
@@ -2,14 +2,10 @@
 
 ```bash
 conda create -y -n dmri_seg python=3.9
```
Comment (RyanZurrin) [inline env_build_commands.md]: dmri_seg needs to be changed to match the version you are using to activate with: dmri-seg

File: env_build_commands.md [hunk 2]
```diff
@@ -2,14 +2,10 @@
...
+conda activate dmri-seg
 pip install tensorflow==2.11
-conda install -y -c anaconda cudnn
-pip install nvidia-pyindex
-pip install nvidia-tensorrt
-conda install -y -c conda-forge nibabel gputil
-pip install scikit-image>=0.16.2
-pip install git+https://github.com/pnlbwh/conversion.git
+conda install -y -c anaconda::cudnn conda-forge::gputil
+pip install nibabel scikit-image git+https://github.com/pnlbwh/conversion.git
```
Comment (RyanZurrin) [inline env_build_commands.md:8]: Is there a specific reason for using pip to install nibabel over conda? I usually try to use conda packages as they resolve dependencies better from my experience. Just wondering if there is a reason behind the change?

**Review by RyanZurrin — APPROVED** [review]

**Review by RyanZurrin — COMMENTED** [review]
Hello Tashrif,

I mistakenly approved the pull request, the only issue is the name of the conda env when it is created needs to be changed as noted in my previous comments.

**Comment by RyanZurrin** [comment]
merged simplified env build commands

**Pull request merged**